### PR TITLE
Bug #15   english pages not yet showing correct info

### PR DESCRIPTION
**Pull request by mpennaste**
* Multilingual links all over the site etc
* Eliminated RU lang

### CODE REVIEW

**Review by marianntapfer — APPROVED** [review]

**Review by LiisKasper — APPROVED** [review]

File: source/films/index.pug [hunk 1]
```diff
@@ -38,12 +38,12 @@ block content
                         else
                             img.container_contents_16_9(src='/assets/img/global/placeholder_hd_1.jpg' alt = val[`filmTitle_${self.locale}`])
                     .card_film_title
-                        h3.film_title_locale= val.filmTitle_et
+                        h3.film_title_locale= val[`filmTitle_${self.locale}`]
```
Comment (mitselek) [inline source/films/index.pug]:
```suggestion
                        - var local_film_title = val[`filmTitle_${self.locale}`]
                        h3.film_title_locale= local_film_title
```

File: source/films/index.pug [hunk 2]
```diff
@@ -38,12 +38,12 @@ block content
...
                     .card_film_info
-                        if val.filmTitle_en
+                        if val[`filmTitle_${self.locale}`]
```
Comment (mitselek) [inline source/films/index.pug]:
```suggestion
                        if local_film_title
```

File: source/films/index.pug [hunk 3]
```diff
@@ -38,12 +38,12 @@ block content
...
                             .str_info_grid_3
                                 .str_info_label= self.label.film.filmTitle_en
-                                .str_info_value= val.filmTitle_en
+                                .str_info_value= val[`filmTitle_${self.locale}`]
```
Comment (mitselek) [inline source/films/index.pug]:
```suggestion
                                .str_info_value= local_film_title
```

**Review by mitselek — CHANGES_REQUESTED** [review]
token.pickle ei tohi siin kaasas olla.

**Pull request merged**